pkg/cmd/operator/sidecar.go

```diff
@@ -118,6 +127,14 @@ func (o *SidecarOptions) Validate() error {
 		}
 	}
 
+	if !helpers.ContainsItem(validation.SupportedBroadcastAddressTypes, scyllav1.BroadcastAddressType(o.NodesBroadcastAddressTypeString)) {
```

Comment:
nit: I guess this could live alongside the time, say `NewBroadcastAddressType(t string) (BroadcastAddressType, error)`

Reply:
With function that converts from string, implementing validation with correct error is harder and there's no point of making it harder.
Compare how webook validation would look like:
```
_, err := scyllav1.NewBroadcastAddressType(string(options.Type))
if err != nil {
    // is it NotSupported or sometheing else happened?
}
```
compared to explicit check:
```
if !helpers.ContainsItem(SupportedBroadcastAddressTypes, options.Type) {
    allErrs = append(allErrs, field.NotSupported(fldPath.Child("type"), options.Type, helpers.ConvertSlice(SupportedBroadcastAddressTypes, helpers.ToString[scyllav1.BroadcastAddressType])))
}
```
The latter one is cleaner, `SupportedBroadcastAddressTypes` is necessary to generate nice error message.